vendor/github.com/docker/docker/api/types/client.go

```diff
@@ -50,7 +50,7 @@ type ContainerCommitOptions struct {
 
 // ContainerExecInspect holds information returned by exec inspect.
 type ContainerExecInspect struct {
-	ExecID      string
+	ExecID      string `json:"ID"`
```

Comment:
do we use this? if so, this will no longer decode in the same way

Reply:
the docker REST API is returning `ID` but the docker client fails to decode it apparently, we don't use or reference this field ourselves.

details in : https://github.com/moby/moby/pull/40475